safe/gui/tools/wizard/test/test_function_centric_wizard.py

```diff
@@ -160,8 +298,8 @@ def test_input_function_centric_wizard(self):
 
         # step_fc_functions1: test number of functions for flood x structure
         dialog.step_fc_functions1.tblFunctions1.setCurrentCell(3, 1)
-        count = len(dialog.step_fc_functions1.selected_functions_1())
-        self.assertEqual(count, expected_flood_structure_functions_count)
+        # count = len(dialog.step_fc_functions1.selected_functions_1())
+        # self.assertEqual(count, expected_flood_structure_functions_count)
```

Comment:
I will remove all legacy test later after we have our new test. Those legacy test will be used as my hint to create the new test.